e2e_tests/tests/api_utils.py

```diff
@@ -24,3 +29,25 @@ def determined_test_session(
         murl, requested_user=credentials.username, password=credentials.password
     )
     return api.Session(murl, credentials.username, authentication.cli_auth, certs.cli_cert)
+
+
+def create_test_user(
+    add_password: bool = False,
+    session: Optional[api.Session] = None,
+    user: Optional[bindings.v1User] = None,
+) -> authentication.Credentials:
+    session = session or determined_test_session(admin=True)
+    user = user or bindings.v1User(username=get_random_string(), admin=False, active=True)
+    password = get_random_string() if add_password else ""
+    bindings.post_PostUser(session, body=bindings.v1PostUserRequest(user=user, password=password))
+    return authentication.Credentials(user.username, password)
+
+
+def log_in_user_cli(credentials: authentication.Credentials) -> None:
```

Comment:
I think the name is a bit confusing, because the `_cli` suffix sounds like you'd be logging in _with the cli_.

Maybe `def configure_token_store()` or something?